src/terminal/terminal/color-scheme.h

```diff
@@ -17,13 +17,13 @@
  * under the License.
  */
 
-#ifndef GUAC_TERMINAL_COLOR_SCHEME_H
-#define GUAC_TERMINAL_COLOR_SCHEME_H
+#ifndef guac_terminal_get_color_scheme_H
+#define guac_terminal_get_color_scheme_H
```

Comment:
The convention for global macros in C is `UPPERCASE_WITH_UNDERSCORES`. I think this may just be the accidental result of a case-insensitive search-and-replace, but this should be `GUAC_TERMINAL_COLOR_SCHEME_H`.

Reply:
Yep, yet another over-agressive search/replace.